contrib/feast/README.md

```diff
@@ -1,16 +1,26 @@
-# Feast Kustomize 
+# Feast Kustomize
 
-## Generating/Updating Feast Kustomize
+## Installing with Kustomize
+
+### Standalone
 
 ```
-kustomize build feast/base | kubectl apply -n kubeflow -f -
+kustomize build feast/base | kubectl apply -n feast -f -
+```
+
+### With Kubeflow
+
+If installing Feast as a component of Kubeflow, use the `kubeflow` overlay.
+
+```
+kustomize build feast/overlays/kubeflow | kubectl apply -f -
```

Comment:
Although not required, adding namespace makes it more consistent. `kustomize build feast/overlays/kubeflow | kubectl apply -n kubeflow -f -`

Reply:
For the Kubeflow overlay every resource will explicitly have a namespace associated, and to keep the install command consistent with the commands for other Kubeflow components listed in the manifests readme, I think it is better to omit the `-n` in the command.